source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Internal/Mappers/ChargeCommandReceivedOutboundMapperTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using GreenEnergyHub.Charges.Domain.ChangeOfCharges.Transaction;
+using GreenEnergyHub.Charges.Domain.Events.Local;
+using GreenEnergyHub.Charges.Infrastructure.Internal.ChargeCommandReceived;
+using GreenEnergyHub.Charges.Infrastructure.Internal.Mappers;
+using GreenEnergyHub.Charges.TestCore;
+using NodaTime;
+using Xunit;
+using Xunit.Categories;
+
+namespace GreenEnergyHub.Charges.Tests.Infrastructure.Internal.Mappers
+{
+    [UnitTest]
+    public class ChargeCommandReceivedOutboundMapperTests
+    {
+        [Theory]
+        [InlineAutoMoqData]
+        public void Convert_WhenCalled_ShouldMapToProtobufWithCorrectValues(
+            [NotNull]ChargeCommand chargeCommand,
+            [NotNull]ChargeCommandReceivedOutboundMapper sut)
+        {
+            // Arrange
+            var guid = Guid.NewGuid().ToString();
+            ChargeCommandReceivedEvent chargeCommandReceivedEvent = new (SystemClock.Instance.GetCurrentInstant(), guid, chargeCommand);
+            UpdateInstantsToValidTimes(chargeCommandReceivedEvent);
```

Comment:
It would read more easily if changed to 
```suggestion
            var chargeCommandReceivedEvent = CreateChargeCommandReceivedEvent(chargeCommand);
```